server/configuration.js

```diff
@@ -54,4 +54,9 @@ module.exports = {
    * example: udp://127.0.0.1
   */
   STATSD_URL: process.env["STATSD_URL"],
+
+  /** Method to use for authorisation of users. Currently accepted values are 'jwt' or ''(none) */
+  AUTH_METHOD: (process.env["AUTH_METHOD"] || ""),
+  /** Secret key for jwt */
+  AUTH_SECRET_KEY: (process.env["AUTH_SECRET_KEY"] || ""),
```

Comment:
Do we need 2 environment variables ? Can we just enforce JWT verification whenever JWT_SECRET_KEY is defined and non-empty ?

Reply:
Yep, fair enough. I included both for any future alternatives, but I've reduced this to 1 in the latest commit